src/base/net/smtp.cpp

```diff
@@ -138,7 +138,7 @@ void Smtp::sendMail(const QString &from, const QString &to, const QString &subje
 {
     const Preferences *const pref = Preferences::instance();
     m_message = "Date: " + getCurrentDateTime().toLatin1() + "\r\n"
-                + encodeMimeHeader(u"From"_qs, from)
+                + encodeMimeHeader(u"From"_qs, tr("qBittorrent %1").arg(from))
```

Comment:
Modern email address needs to be enclosed in angle brackets
https://datatracker.ietf.org/doc/html/rfc5322#section-3.4

```suggestion
                + encodeMimeHeader(u"From"_qs, tr("qBittorrent <%1>").arg(from))
```

Reply:
Thanks, it is, done